.github/workflows/tf-plan-hash.yml

```diff
@@ -0,0 +1,130 @@
+name: HASH Terraform Plan
+
+# TODO: While this workflow is specific to HASH, we should consider
+#  making this a reusable workflow, parameterized on the directory and environment.
+
+on:
+  workflow_dispatch:
+    inputs:
+      env:
+        type: choice
+        options: ["prod"]
+        description: "Environment to plan"
+        required: true
+        default: "prod"
+
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "infra/terraform/hash/**"
+      - "infra/terraform/modules/**"
+
+env:
+  VAULT_ADDR: ${{ secrets.VAULT_ADDR }}
+
+jobs:
+  matrix-eval:
+    name: Prepare environments to be planned
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Set matrix
+        id: set-matrix
+        env:
+          ENV: ${{ github.event.inputs.env }}
+        run: |
+          # If no env is specified, run both dev and prod
```

Comment:
Either this comment needs correcting or the matrix below does

Reply:
Addressed in f21e71353927c3f6af984d7b4111acf9fd9d3db7